.devcontainer/scripts/postCreate.sh

```diff
@@ -0,0 +1,40 @@
+# Set up git
+git config --global --add safe.directory /workspaces/dbt-synthea
+git config --global init.defaultBranch main
+
+# Install requirements
+cd /workspaces/dbt-synthea
+python -m venv dbt-env
+source dbt-env/bin/activate 
+pip install -r .devcontainer/scripts/minimal_requirements.txt
```

Comment:
Agree - a separate virtual environment within a devcontainer is redundant!

But, the python interpreter path has been hardcoded in [.vscode/settings.json](https://github.com/vvcb/dbt-synthea/blob/vc/databricks/.vscode/settings.json#L7) currently. This causes an issue with `dbt-power-user` extension not picking up the correct python environment.

Reply:
Regarding _no advantage in locking version_, certainly have had bad experience with the `dbt-sqlserver` adapter that broke the environment on more than one occasion by installing incompatible versions of dbt-core and dbt-sqlserver. But this was over six months ago and the issues may have been fixed since. 

Never a bad idea for locking package versions in a devcontainer. Someone working on a devcontainer created 3 months ago could easily be working with a different environment to what someone creates in 3 months time from the same code base.

Reply:
This is true ~ it's more an issue of python not having a standard way of allowing [deterministic builds](https://peps.python.org/pep-0751/) in my opinion (dumping `pip freeze` isn't a maintainable solution), but this is again out of the scope of this PR